driver/gamepad.c

```diff
@@ -293,11 +288,14 @@ static int gip_gamepad_op_input(struct gip_client *client, void *data, u32 len)
 	{
 		u8 paddles;
 		struct gip_firmware_version *firmw_ver = (struct gip_firmware_version *)client->firmware_versions->data;
-		if(firmw_ver->major == 5 && firmw_ver->minor >= 11) {
+		if(firmw_ver->major >= 5 && firmw_ver->minor >= 11)
+		{
 			//I know this is not ideal...
 			unsigned char *data_bytes = (unsigned char *)data;
 			paddles = data_bytes[64];
-		} else if(firmw_ver->major == 4) {
+		}
+		else
```

Comment:
If it fails it fails, and it needs to be fixed anyways, so I would assume it "might" work for now.